src/index.js

```diff
@@ -1,29 +1,76 @@
 import * as C from './constant'
-import * as Utils from './utils'
+import * as U from './utils'
+import en from './locale/en'
 
-const parseConfig = (config) => {
+let L = 'en' // global locale
+const Ls = {} // global loaded locale
```

Comment:
Every time someone adds a locale here it is permanently stored by dayjs, conceivably people could do

``` js
const process = (obj) => {
  // some logic here
  const locale = generate_locale(obj);
  const time = dayjs();

  return time.locale(custom_locale);
}
```
If this function is called many times and each time a unqiue local is generated (unlikely I know but possible) then dayjs will end up storing all the locales indefinitely.

This would be a good place to use a weak map: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap

Reply:
Nice point, but I don't really understand (sad face). A little more detail, please?

Reply:
I will make a PR with a fix once this is merged, might be simpler to think about it as a seperate issue.